rules/osquery_rules/osquery_linux_mac_vulnerable_xz_liblzma.py

```diff
@@ -0,0 +1,52 @@
+# SELECT 'Homebrew Package' AS source, name, version,
+#   CASE
+#     WHEN version LIKE '5.6.0%' OR version LIKE '5.6.1%' THEN 'Potentially Vulnerable'
+#     ELSE 'Most likely not vulnerable'
+#   END AS status
+# FROM homebrew_packages
+# WHERE name = 'xz' OR name = 'liblzma';
+
+# SELECT 'DEB Package' AS source, name, version,
+#   CASE
+#     WHEN version LIKE '5.6.0%' OR version LIKE '5.6.1%' THEN 'Potentially Vulnerable'
+#     ELSE 'Most likely not vulnerable'
+#   END AS status
+# FROM deb_packages
+# WHERE name = 'xz-utils' OR name = 'xz-libs' OR name = 'liblzma' OR name LIKE 'liblzma%'
+# UNION
+# SELECT 'RPM Package' AS source, name, version,
+#   CASE
+#     WHEN version LIKE '5.6.0%' OR version LIKE '5.6.1%' THEN 'Potentially Vulnerable'
+#     ELSE 'Most likely not vulnerable'
+#   END AS status
+# FROM rpm_packages
+# WHERE name = 'xz-utils' OR name = 'xz-libs' OR name = 'liblzma' OR name LIKE 'liblzma%';
+
+VULNERABLE_PACKAGES = {"xz", "liblzma", "xz-libs", "xz-utils"}
+VULNERABLE_VERSIONS = {"5.6.0", "5.6.1"}
+
+
+def rule(event):
+    if event.get("action") != "added":
+        return False
+    name = event.deep_get("columns", "name", default="")
```

Comment:
In the past, I typically also filtered on the osquery query "name" in the top-level to make sure we were analyzing the right output. You don't need to do it, but it would help make the rule more accurate. You could use the default pack query names as a reference.

Reply:
https://github.com/osquery/osquery/tree/master/packs

Reply:
Makes sense, I'll adjust the rule so it looks at the default pack queries and doesn't require custom queries to be added.